ngraph/python/tests/conftest.py

```diff
@@ -25,11 +25,17 @@ def pytest_addoption(parser):
         choices=["CPU", "GPU", "FPGA", "HDDL", "MYRIAD", "HETERO"],
         help="Select target device",
     )
+    parser.addoption(
+        "--additional_models",
+        default="",
+        type=str,
+    )
 
 
 def pytest_configure(config):
     backend_name = config.getvalue("backend")
     tests.BACKEND_NAME = backend_name
+    tests.ADDITIONAL_MODELS_DIR = config.getvalue("additional_models")
```

Comment:
Not needed.

Reply:
Needed with configurable path to additional models path.